main.js

```diff
@@ -77,20 +77,20 @@ const log = report => {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 };
 
+const files = (report, predicate) => report.results
+	.filter(predicate)
```

Comment:
@schnittstabil Some feedback would be appreciated :)

For 0, I think we can assume the counts are always numbers (and should they be `undefined` they will be filtered out). See [index.L10](https://github.com/sindresorhus/xo/blob/master/index.js#L10) and [index.L68](https://github.com/sindresorhus/xo/blob/master/index.js#L68)
Are my assumptions good? Should the warnings only be opened if they are no erros?

I would prefer using the descending sorter, that way the file with the most errors will be opened first.

Thoughts?